### PR TITLE
Add 4.4 beta repositories

### PR DESCRIPTION
**Pull request by mnecas**
Add 4.4 beta repositories 
@mwperina @dangel101

### CODE REVIEW

File: tasks/rh-subscription.yml [hunk 1]
```diff
@@ -61,4 +61,4 @@
   command: "dnf module enable -y {{ ovirt_repositories_rh_dnf_modules | join(' ') }}"
   args:
     warn: False
-  when: ovirt_repositories_ovirt_version >= 4.4
+  when: ovirt_repositories_ovirt_version|string >= '4.4'
```
Comment (mnecas) [inline tasks/rh-subscription.yml:64]: Another solution can be `(ovirt_repositories_ovirt_version|string).split('-')[0]|float >= 4.4` but will have same result

File: tasks/rpm.yml [hunk 2]
```diff
@@ -8,4 +8,4 @@
   command: "dnf module enable -y {{ ovirt_repositories_ovirt_dnf_modules | join(' ') }}"
   args:
     warn: False
-  when: ovirt_repositories_ovirt_version >= 4.4
+  when: ovirt_repositories_ovirt_version|string >= '4.4'
```
Comment (mnecas) [inline tasks/rpm.yml:11]: Same here

File: vars/engine_4.4-beta.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+ovirt_repositories_subscription_manager_repos:
+  - rhel-8-for-x86_64-baseos-rpms
+  - rhel-8-for-x86_64-appstream-rpms
+  - rhv-4-manager-beta-for-rhel-8-x86_64-rpms
+  - rhv-4-tools-beta-for-rhel-8-x86_64-rpms
+  - ansible-2.9-for-rhel-8-x86_64-rpms
+  - jb-eap-7.2-for-rhel-8-x86_64-rpms
```
Comment (mwperina) [inline vars/engine_4.4-beta.yml]: 7.3

**Review by mwperina — CHANGES_REQUESTED** [review]

**Review by mwperina — APPROVED** [review]

**Review by mwperina — APPROVED** [review]
+1

File: tasks/rh-subscription.yml [hunk 4]
```diff
@@ -55,10 +55,11 @@
   rhsm_repository:
     name: "{{ item }}"
     state: "enabled"
+  ignore_errors: True
```
Comment (mwperina) [inline tasks/rh-subscription.yml]: Instead of ignoring errors I'd remove tools channel. Patch for RHV docs removing tools channel should be merged today

Comment (mnecas) [reply tasks/rh-subscription.yml]: Okay, will update it merge it and do the build so we can have it for the demo.

**Pull request merged**